sample/src/main/java/com/alexstyl/contactstore/sample/ContactListViewModel.kt

```diff
@@ -26,7 +27,7 @@ class ContactListViewModel @Inject constructor(
             reloadContacts.collect {
                 contactStore.fetchContacts()
                     .collect {
-                        state.emit(Loaded(it))
+                        state.emit(Loaded(it.sortedByDescending { contact -> contact.isStarred }))
```

Comment:
I like the idea of this, but given there is no visual indicator in the UI why some contacts are on top might be confusing.

Let's remove it unless you feel strongly about this.

Reply:
Yeah, makes sense. Maybe future contributors will add different sections in the list